SingularityUI/app/actions/api/base.es6

```diff
@@ -70,14 +48,46 @@ export function buildApiAction(actionName, opts = {}) {
     return { type: STARTED };
   }
 
-  function error(error) {
-    return { type: ERROR, error };
+  function error(ex) {
+    return { type: ERROR, ex };
```

Comment:
this changes the action's error key from `error` to `ex` -- was this intentional? is anything depending on it being `error` yet?